README.rst

```diff
@@ -155,3 +155,75 @@ Using your normal views over a websocket connection
       ),
    })
 
+
+Creating fully-functional custom Consumers
+------------------------------------------
+
+This package offers Django Rest Framework capabilities via mixins. To utilize these mixins, one must inherit from the GenericAsyncAPIConsumer. One may use the same exact querysets and serializer_classes utilized in their DRF Views. However, one must import djangochannelsrestframework's permissions, which provides the standard AllowAny and IsAuthenticated permissions.
+
+from . import models
```

Comment:
for this code block look above 

```
.. code-block:: python
```

note how the code is indeneted one level as well.